style.css

```diff
@@ -0,0 +1,1086 @@
+  color: white;
+  font-size: 2.4em;
+  font-weight: 800;
+  width: 90%;
+  margin: 0;
+  padding: 0;
+  padding: 0 10px;
+  z-index: 2;
+}
+
+.sub-img-text {
+  margin-top: 15%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  z-index: 2;
+  padding: 10px 15px;
+}
+
+.sub-img-text h3 {
+  color: white;
+  font-weight: 700;
+  font-size: 1.5em;
+  z-index: 2;
+  margin: 0;
+  padding: 0;
+}
+
+.sub-img-text > div {
+  z-index: 10;
+}
+
+.sub-img-text small {
+  z-index: 10;
+  padding-right: 10px;
+}
+
+.sub-img-text time {
+  z-index: 10;
+  color: white;
+}
+
+.sub-img-text > div small {
+  color: white;
+}
+
+.lastest-elements small {
+  font-size: 0.8em;
+  font: inherit;
+  color: #555;
+}
+
+.cover-text .byline small {
+  color: white;
+  padding: 10px;
+}
+
+.header-img .img-overlay {
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  position: absolute;
+  opacity: 0.6;
+  z-index: 1;
+  background: rgba(255, 211, 209, 0.74);
+  background:
+    -moz-linear-gradient(
+      -45deg,
+      rgba(255, 211, 209, 0.74) 0%,
+      rgba(240, 12, 0, 0.86) 51%,
+      rgba(0, 0, 0, 0.98) 100%
+    );
+  background:
+    -webkit-gradient(
+      left top,
+      right bottom,
+      color-stop(0%, rgba(255, 211, 209, 0.74)),
+      color-stop(51%, rgba(240, 12, 0, 0.86)),
+      color-stop(100%, rgba(0, 0, 0, 0.98))
+    );
+  background:
+    -webkit-linear-gradient(
+      -45deg,
+      rgba(255, 211, 209, 0.74) 0%,
+      rgba(240, 12, 0, 0.86) 51%,
+      rgba(0, 0, 0, 0.98) 100%
+    );
+  background:
+    -o-linear-gradient(
+      -45deg,
+      rgba(255, 211, 209, 0.74) 0%,
+      rgba(240, 12, 0, 0.86) 51%,
+      rgba(0, 0, 0, 0.98) 100%
+    );
+  background:
+    -ms-linear-gradient(
+      -45deg,
+      rgba(255, 211, 209, 0.74) 0%,
+      rgba(240, 12, 0, 0.86) 51%,
+      rgba(0, 0, 0, 0.98) 100%
+    );
+  background:
+    linear-gradient(
+      135deg,
+      rgba(255, 211, 209, 0.74) 0%,
+      rgba(240, 12, 0, 0.86) 51%,
+      rgba(0, 0, 0, 0.98) 100%
+    );
+}
+
+.wrapper {
+  width: 80vw;
+  margin: 0 auto;
+}
+
+.wrapper h3 {
+  font-family: inherit;
+  color: rgba(248, 80, 50, 1);
+  font-size: 2.5em;
+}
+
+.lastest-news {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.lastest-elements {
+  width: 24%;
+
+  color: black;
+}
+
+.lastest-elements img {
+  width: 100%;
+  height: 170px;
+}
+
+.lastest-elements h5 {
+  font-size: 1em;
+}
+
+.latest-funding {
+  width: 85vw;
+  margin: 0 auto;
+}
+
+.latest-funding .section-header-text {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.latest-funding .section-header-text h5 {
+  color: #f42;
+  font-size: 1.5em;
+}
+
+.latest-funding .section-header-text a {
+  text-decoration: none;
+}
+
+.latest-section-wrapper .header-link a {
+  text-decoration: none;
+  color: #f42;
+}
+
+.latest-funding .section-header-text .links a {
+  margin-left: 10px;
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.


**Example:**
```CSS
/* before */
.latest-funding .section-header-text .links a {

/* after */
.anchortag-class {
```
And the same approach for the previous in this highlighted box 
<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_